tests/eas/load_tracking.py

```diff
@@ -776,7 +804,8 @@ def _test_util_per_cpu(self, experiment, tasks):
                 cpus.update(self._get_cpus(task[1], phase))
 
             # Get the mean utilization per CPU
-            util_mean = self._get_util_mean(window[phase][0], window[phase][1],
+            util_mean = self._get_util_mean(experiment, phase,
```

Comment:
As a general comment, I would say that at the end we are interested in the overall pct of creepy tasks showing up in the entire trace and I also don't know how much interest we can have in the phase info.

Thus, I would say that we could consider to avoid entirely to pass `experiment` and `phase` to  `_get_util_mean` and run our sanity checks even before computing the mean. Consider that, if the mean is within margins but we have an high noise level... still we cannot trust it and the test should fail anyway.

What do you think?

Reply:
I think we will lose a lot of accuracy in this case.

For example if we have 2 phases and a noise of 0.3% creeps into each phase, it's overall small for each phase and the test should run smoothly. If we have a noise margin of 0.6% each phase will be happy to deal with that. But the combined noise is 0.6% which can look high and we could prematurely decide it's a failure.

Similarly if we increase the margin to be for instance 0.9% since we are looking at the noise value in the entire test duration, then if we have a per phase noise levels of 0.6% + 0.2% then the overall test will say things are OK, but 0.6% at phase one is too high and will cause false negatives.

Makes sense?